gulpfile.js

```diff
@@ -0,0 +1,54 @@
+// Include gulp
+var gulp = require('gulp');
+
+// Include Our Plugins
+var jshint = require('gulp-jshint');
+var sass = require('gulp-sass');
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+var rename = require('gulp-rename');
+var inlinesource = require('gulp-inline-source');
+var autoprefixer = require('gulp-autoprefixer');
+
+// Lint Task
+gulp.task('lint', function() {
+  return gulp.src('js/*.js')
+  .pipe(jshint())
+  .pipe(jshint.reporter('default'));
+});
+
+// Compile Our Sass
+gulp.task('sass', function() {
+  return gulp.src('src/scss/*.scss')
+  .pipe(sass())
+  .pipe(autoprefixer({
+    browsers: ['last 2 versions'],
+    cascade: false
+  }))
+  .pipe(gulp.dest('src/css'));
+});
+
+// Concatenate & Minify JS
+gulp.task('scripts', function() {
+  return gulp.src('js/*.js')
+  .pipe(concat('all.js'))
+  .pipe(gulp.dest('dist'))
+  .pipe(rename('all.min.js'))
```

Comment:
@christianv sorry need some clarification. should we remove jshint and jscs files even when we are using gulpfile.js?